modules/clients/src/test/java/org/apache/ignite/internal/TaskEventSubjectIdSelfTest.java

```diff
@@ -139,7 +139,7 @@ public void testSimpleTask() throws Exception {
         assert evt != null;
 
         assertEquals(EVT_TASK_STARTED, evt.type());
-        assertEquals(nodeId, evt.subjectId());
```

Comment:
I don't get what this test is currently testing? Description says that it should test subject Id, but currently it checked that all subject ids is null. Perhaps we should enable security for this test and revert `evt.subjectId()` checking.

Reply:
I removed this test - the same test cases are included in the ComputeTaskRemoteSecurityContextTest, but it is more comprehensive.